assets/src/components/form/select/model-selects/status.js

```diff
@@ -27,14 +27,20 @@ export default class StatusSelect extends Component {
 		},
 		...statusModel.defaultQueryData,
 		getQueryString: statusModel.getQueryString,
-		selectedType: 'registration',
 		selectLabel: __( 'Select Status', 'event_espresso' ),
 		addAllOptionLabel: __( 'All Statuses', 'event_espresso' ),
 	};
 
 	static propTypes = {
 		...statusModel.queryDataTypes,
-		selectedType: PropTypes.string.isRequired,
+		statusType: PropTypes.oneOf( [
+			statusModel.STATUS_TYPE_EMAIL,
+			statusModel.STATUS_TYPE_EVENT,
+			statusModel.STATUS_TYPE_MESSAGE,
+			statusModel.STATUS_TYPE_PAYMENT,
+			statusModel.STATUS_TYPE_REGISTRATION,
+			statusModel.STATUS_TYPE_TRANSACTION,
+		] ),
```

Comment:
You probably should also append `isRequired` to this as well UNLESS it's okay if its not included.

Reply:
ohh ya good catch... will do